apps/web/app/(app)/onboarding/page.tsx

```
@@ -1,84 +1,72 @@
+      </TypographyP>
 
-            <SubmitButtonWrapper>
-              <div className="ml-auto">
-                <Button>Next</Button>
-              </div>
-            </SubmitButtonWrapper>
-          </div>
-        </Card>
+      <div className="my-8">
+        <Steps
+          selectedStep={parseInt(searchParams.step || "1")}
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)